vent/types/sql_supported_databases.go

```diff
@@ -0,0 +1,7 @@
+package types
+
+// supported databases
+const (
+	PostgresDB = "postgres"
+	SQLite     = "sqlite"
```

Comment:
Just for consistency, can you rename either `PostgresDB` to `Postgres`, or `SQLite` to `SQLiteDB`?

Reply:
ok